amplify/src/main/java/com/github/stkent/amplify/tracking/predicates/LastTimePredicate.java

```diff
@@ -40,4 +40,9 @@ public void eventTriggered(@NonNull IEvent event) {
             updateEventValue(event, currentTime);
         }
     }
+
+    @Override
+    public Long defaultValue() {
+        return 0L;
```

Comment:
No, wait, max value should work for both, right? We'll look at the value of T = currentTimeMillis - retrieved value, and for warmup check:

T > threshold

and for cooldown check

`T < 0 || T > threshold`?